app/definitions/IRoom.ts

```diff
@@ -55,13 +52,11 @@ export enum OmnichannelSourceType {
 	API = 'api',
 	OTHER = 'other' // catch-all source type
 }
-export interface IOmnichannelRoom extends Omit<IRoom, 'default' | 'featured' | 'broadcast' | ''> {
+export interface IOmnichannelRoom extends Partial<Omit<IRoom, 'default' | 'featured' | 'broadcast' | ''>> {
```

Comment:
I dont got why the `| ' ' > ` on the end of the line

Reply:
Missclick. Fixed!